rapids/verify_rapids_spark.py

```diff
@@ -0,0 +1,25 @@
+from pyspark.sql import SparkSession
+from pyspark import SparkConf, SparkContext
+
+conf = SparkConf().setAppName("NYTaxiXGBoost4j")
+conf.set("spark.executor.instances", "1")
+conf.set("spark.executor.cores", "1") # spark.executor.cores times spark.executor.instances should equal total cores.
+conf.set("spark.task.cpus", "1")
+conf.set("spark.executor.memory", "2g")
+conf.set("spark.task.resource.gpu.amount", "1")
+conf.set("spark.plugins", "com.nvidia.spark.SQLPlugin")
```

Comment:
We can probably move line 9 into the initialization action, and line 10 is already included there. Perhaps this will help fix the Spark 2 issues?

Reply:
made some changes and make sure it works in both spark2.x and spark 3.